tests/unit/remote/ssh/test_ssh.py

```diff
@@ -1,5 +1,4 @@
 import getpass
-
```

Comment:
Let's not remove this line, since it separates test modules from getpass.

Reply:
DONE.